synapse/models/base.py

```diff
@@ -257,81 +232,5 @@ def getModelDefs(self):
                         'doc': 'An external identifier for the rule.'}),
                 )),
 
-                ('graph:cluster', {}, (
```

Comment:
Have we marked these as deprecated model elements in master yet?

Reply:
There's a separate PR up for that https://github.com/vertexproject/synapse/pull/3674